Sources/DocCDocumentation/DocCDocumentation.docc/Purchases.md

```diff
@@ -77,6 +77,10 @@ Most features require configuring the SDK before using it.
 - ``Purchases/showManageSubscriptions()``
 - ``Purchases/showManageSubscriptions(completion:)``
 
+### Parsing Receipts
+- ``PurchasesReceiptParser``
+- ``AppleReceipt``
```

Comment:
We are planning to move these to the SPM library later right? I'm concerned about making this part of the SDK public API here and then moving them to a library later on since that would mean it would be a breaking change unless we can somehow make it a transitive dependency... Should we hold on merging this to main until the SPM library part is also ready? (I haven't checked #2155 yet)

Reply:
See my other comment, we’ll be exposing this as part of the SDK too for users who already use the main Framework.

Reply:
Ahhh ok makes sense!